.github/workflows/uc-crux-llvm-lint.yaml

```diff
@@ -16,11 +16,16 @@ jobs:
 
       - shell: bash
         run: |
-          cd uc-crux-llvm/
           curl --location -o hlint.tar.gz \
             https://github.com/ndmitchell/hlint/releases/download/v3.3/hlint-3.3-x86_64-linux.tar.gz
           tar xvf hlint.tar.gz
-          ./hlint-3.3/hlint exe src test
+
+          pushd crucible-llvm-syntax/
+          ../hlint-3.3/hlint src test
+          popd
```

Comment:
I usually recommend using the `(cd tgtdir; something)` syntax over a `pushd tgtdir; something; popd` because it's easier to get the pushd/popd pairs mismatched.